common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1296,8 +1296,9 @@
     public static final KnownKey zm_oauth_classes_handlers_yahoo = KnownKey.newKey("com.zimbra.oauth.handlers.impl.YahooOAuth2Handler");
     public static final KnownKey zm_oauth_classes_handlers_google = KnownKey.newKey("com.zimbra.oauth.handlers.impl.GoogleOAuth2Handler");
     public static final KnownKey zm_oauth_classes_handlers_facebook = KnownKey.newKey("com.zimbra.oauth.handlers.impl.FacebookOAuth2Handler");
-    
-    
+    public static final KnownKey zm_oauth_classes_handlers_linkedin = KnownKey.newKey("com.zimbra.oauth.handlers.impl.LinkedinOAuth2Handler");
```

Comment:
Prashant said we were going to put LinkedIn aside for now?

If we're adding them all, please add twitter.